api/src/main/scala/hmda/api/http/institutions/UploadPaths.scala

```diff
@@ -81,7 +84,9 @@ trait UploadPaths extends InstitutionProtocol with ApiErrorProtocol with Submiss
         val uploadedF = byteSource
           .via(splitLines)
           .map(_.utf8String)
-          .runForeach(line => processingActor ! AddLine(uploadTimestamp, line))
+          //.runForeach(line => processingActor ! AddLine(uploadTimestamp, line))
+          .mapAsync(parallelism = flowParallelism)(line => (processingActor ? AddLine(uploadTimestamp, line)).mapTo[Persisted])
```

Comment:
I think we **do** care though. If we are ever asked to **exactly** reproduce the file that was sent to us (this has come up in the past), having things in order is going to be mandatory. In addition to this, `mapAsyncUnordered` is a nice optimization, but we don't know at this point if / how much it really helps